apple/internal/ios_rules.bzl

```diff
@@ -186,6 +187,97 @@ def _ios_application_impl(ctx):
         binary_target[apple_common.AppleExecutableBinary],
     ] + processor_result.providers
 
+def _ios_app_clip_impl(ctx):
+    """Experimental implementation of ios_app_clip."""
+    top_level_attrs = [
+        "app_icons",
+        "launch_images",
+        "launch_storyboard",
+        "strings",
+        "resources",
+    ]
+
+    # TODO(kaipi): Replace the debug_outputs_provider with the provider returned from the linking
```

Comment:
I like how it appears as I'm adding newer todos to this code :p

(you can just remove the `(kaipi)` part since I'm not going to be taking a look at it any time soon :) )